pytest_httpx/_httpx_mock.py

```diff
@@ -15,19 +16,22 @@ def __init__(
         method: Optional[str] = None,
         match_headers: Optional[Dict[str, Any]] = None,
         match_content: Optional[bytes] = None,
+        match_json_content: Optional[Any] = None,
     ):
         self.nb_calls = 0
         self.url = httpx.URL(url) if url and isinstance(url, str) else url
         self.method = method.upper() if method else method
         self.headers = match_headers
         self.content = match_content
+        self.json_content = match_json_content
```

Comment:
We should add some logic to prevent user from providing boths content and json here. I don't see a use case where it would make sense.

Reply:
Yeah, agreed.

Reply:
I am raising a ValueError. Does that make sense?